### ISSUE TITLE
Replicating results from paper with dropout

### ISSUE THREAD

**Issue by spaudel** [wjmaddox/swa_gaussian#10]
Hi, I am trying to replicate the results from your paper, and I was unsure how you trained the model with swag-dropout. I tried to use the same command script as from readme, and replace the model with dropout - but the losses do not converge. Could you describe how to run an experiment for image classification and get results for MCMC dropout.

**Comment by wjmaddox** [comment]
Hi,

What command did you attempt to run and with which model?

I believe that we simply turned the dropout layers back on at test time for test MC Dropout.

Please see https://github.com/wjmaddox/swa_gaussian/blob/master/experiments/uncertainty/README.md for an example command.

**Comment by spaudel** [comment]
Hi Wesley,
Thanks for responding. That is the instruction I was following. My question was more for training the dropout models. I tried training vgg dropout model with the following command

`python experiments/train/run_swag.py --dir=./results/vgg_dropout_cifar10 --dataset=CIFAR10 --data_path=./datasets/ --model=VGG16Dropout --epochs=300 --lr_init=0.05`

This model never converged, the losses started increasing, and the accuracy started dropping after about 50 epochs or so. 

I think this is because I am not training dropout models properly, and wanted to check with you regarding your commandline parameters.

**Comment by wjmaddox** [comment]
Looks like you forgot to set the weight decay properly `wd=5e-4` It doesn't seem to be diverging for me.

**Comment by spaudel** [comment]
Thanks a lot for the quick response. The network is converging now.
